tests/unittests/tests/01-run.py

```diff
@@ -7,13 +7,17 @@
 # directory for more details.
 
 import os, signal, sys, subprocess
-from pexpect import spawn, TIMEOUT, EOF
+import pexpect
 
 DEFAULT_TIMEOUT = 5
 
 def main():
     env = os.environ.copy()
-    child = spawn("make term", env=env, timeout=DEFAULT_TIMEOUT)
+    if sys.version_info < (3,):
```

Comment:
you can pass the `encoding` to `spawn` and enforce unicode. There's no need for `spawnu`.